pkg/feeds/packagist/packagist_test.go

```diff
@@ -29,13 +29,17 @@ func TestFetch(t *testing.T) {
 	feed.versionHost = srv.URL
 
 	cutoff := time.Unix(1614513658, 0)
-	latest, errs := feed.Latest(cutoff)
+	latest, gotCutoff, errs := feed.Latest(cutoff)
 	if len(errs) != 0 {
 		t.Fatalf("got error: %v", errs[len(errs)-1])
 	}
 	if len(latest) == 0 {
 		t.Fatalf("did not get any updates")
 	}
+	wantCutoff := time.Date(2021, 2, 28, 12, 20, 3, 0, time.UTC)
```

Comment:
as above - is the unix time related here?

Reply:
Done.

The unix time above is not really related. I'm not sure why its used in this case.